paper/paper.md

```diff
@@ -0,0 +1,180 @@
+---
+title: 'Pyglmnet: Python implementation of elastic-net regularized generalized linear models'
+tags:
+  - Python
+  - glm
+  - machine-learning
+  - lasso
+  - elastic-net
+  - group-lasso
+authors:
+  - name: Mainak Jas
+    orcid: 0000-0003-0872-7098
+    affiliation: "1, 2"
+  - name: Titipat Achakulvisut
+    affiliation: 3
+  - name: Aid Idrizović
+    affiliation: 4
+  - name: Daniel Acuna
+    affiliation: 5
+  - name: Matthew Antalek
+    affiliation: 6
+  - name: Vinicius Marques
+    affiliation: 4
+  - name: Tommy Odland
+    affiliation: 7
+  - name: Ravi Prakash Garg
+    affiliation: 6
+  - name: Mayank Agrawal
+    affiliation: 8
+  - name: Yu Umegaki
+    affiliation: 9
+  - name: Peter Foley
```

Comment:
My name and affiliation look good, and here's a code suggestion with my orcid. Thanks!
```suggestion
  - name: Peter Foley
  - orcid: 0000-0002-0304-7213
```